SparkPost/matchbox
# UX-295 Zebra stripes begin on odd rows

### What Changed
- Uses `odd` instead of `even` in the zebra striping style function


### How To Test or Verify
- Run storybook
- Visit any of the table stories to verify styling


### PR Checklist
